src/DependencyInjection/Configuration.php

```diff
@@ -50,15 +51,33 @@ public function getConfigTreeBuilder()
                 ->arrayNode('indices')
                     ->useAttributeAsKey('name')
                     ->arrayPrototype()
+                        ->beforeNormalization()
+                            ->ifTrue(function($v) {
+                                return !empty($v['serializer_groups']) &&
+                                       (!isset($v['enable_serializer_groups']) || !$v['enable_serializer_groups']);
+                            })
+                            ->thenInvalid('In order to specify "serializer_groups" you need to enable "enable_serializer_groups"')
+                        ->end()
```

Comment:
I don't see a reason why this node should (or can it even?) get a default value of `false`?
This is just to prevent putting custom groups and wondering why this doesn't work (so really an DX improvement here).